introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-llama-2-chat.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>Do you want add the default instance type and max token information clearly. May be some information on how to change default instance type as in falcon or just pointing to jumpstart model documentation </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Added a section on  changing instance type. The total number of tokens is described in the notes section below.</p>